### DISCUSSION TITLE
Jetstream KV with TTL

### DISCUSSION THREAD

**Discussion by alberk8** [nats-io/nats-server#5583] [General]
I would like to know if it is possible for KV with TTL to get TTL Deleted in the KV Watch?   Is it a configuration or not implemented feature?  Thank you.

**Comment by derekcollison** [comment]
This is coming in 2.11.

We will have per msg (key) TTL, the ability to do LRU, and system delete events for watchers.

/cc @ripienaar

**Reply by alberk8** [reply]
Thank you for the prompt reply.
